snakemake/logging.py

```
@@ -305,12 +243,504 @@
+            "progress": "progress",
+            "shellcmd": "progress",  # or other quietness types
+            "job_finished": "progress",
+            "resources_info": "progress",
+            "run_info": "progress",
+            "host": "host",
+            "info": "progress",
+        }
+
+        # Check quietness for specific levels and skip accordingly
+        if level in quietness_map:
+            if self.is_quiet_about(quietness_map[level]):
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)